server/routes/index.js

```diff
@@ -1,8 +1,11 @@
 var express = require("express");
 var router = express.Router();
+const projectRoutes = require('./Project');
 
-router.get("/welcome", function(req, res, next) {
+router.get("/welcome", function (req, res, next) {
   res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
 });
 
+router.use('/projects', projectRoutes)
+
```

Comment:
Related to your auth routes comment -- will the plan be to just have a routes file for each model and add the protected route middleware to each route?

Reply:
yeah!